src/main/java/address/controller/MainController.java

```diff
@@ -559,4 +545,10 @@ public void stop() {
         getPrimaryStage().hide();
         releaseResourcesForAppTermination();
     }
+
+    private void showAlertDialogWithAssertion(String title, String headerText, String contentText, String errorLocation) {
```

Comment:
Name is too general, this is specifically for fxml load failures right?
How about something like `handleFatalFxmlLoadError`?